composer.json

```diff
@@ -23,8 +23,8 @@
     },
     "require": {
         "php": ">=5.4.0",
-        "doctrine/orm": ">=2.2.1,<2.7-dev",
-        "doctrine/dbal": ">=2.2.1,<2.7-dev",
+        "doctrine/orm": ">=2.2.1,<3.0-dev",
```

Comment:
For reference, see [`composer.json`](https://github.com/doctrine/doctrine2/blob/12b28571f55e4f04e3d480857fe79726d3e7dbb1/composer.json#L46-L50) of `doctrine/orm:dev-master`.